src/MusicTheory/Duration.elm

```diff
@@ -0,0 +1,37 @@
+module MusicTheory.Duration exposing
```

Comment:
I know this is still WIP. I more and more tend to make things opaque by default, and only not do it if there is a good reason for it.

Reply:
Yes, I will definitely do that.